WordPress/src/main/res/values/strings.xml

```diff
@@ -1307,7 +1307,7 @@
     <string name="reader_filter_empty_tags_action">Add a tag</string>
     <string name="reader_filter_self_hosted_empty_sites_tags_action">Log in to WordPress.com</string>
     <string name="reader_dot_separator" translatable="false">\u0020\u2022\u0020</string>
-    <string name="reader_error_generic_title">There was a problem handling the request. Please try again later.</string>
+    <string name="reader_error_request_failed_title">There was a problem handling the request. Please try again later.</string>
```

Comment:
This change will remove all the translation in GlotPress. We might want to consider creating a new untranslatable key and referencing the old key. Wdyt?

Reply:
Old key `reader_error_generic_title` was added only on the Interests Picker screen which is still not a live feature. I just renamed it. Wondering if we still need to fix it.

Reply:
If it was in develop before the code freeze the strings were already uploaded to GlotPress and translated to at least the main languages. Having said that I guess it's not a big deal. We can just rename it.